File: Basic Programming/Hackerrank 30 days of code python3 solution/Day_6.py
#                      Name : Arif Khan
#                      Judge: HACKERRANK
#                      University: Primeasia University
#                      problem: Day 6: Let's Review
#                      Difficulty: Easy
#                      Problem Link: https://www.hackerrank.com/challenges/30-review-loop/problem
#


n = int(input())
while n != 0:
    s = input()
    w1, w2 = "", ""
    for i in range(0, len(s)):
        if i%2:
            w2 += s[i]
        else:
            w1 += s[i]
    print("{} {}".format(w1, w2))
    n -= 1

File: Basic Programming/Hackerrank 30 days of code c++ solution/Day_6.cpp
/*                    Name : Arif Khan
                      Judge: HACKERRANK
                      University: Primeasia University
                      problem: Day 6: Let's Review
                      Difficulty: Easy
                      Problem Link: https://www.hackerrank.com/challenges/30-review-loop/problem
*/

#include <bits/stdc++.h>
using namespace std;


int main() {
    string a;
    char b[100000],c[100000];
    int i,n,k,j,l;
    cin>>n;
    while(n>0){
        cin >> a ;
        l = a.size();
        j = k = 0;
        for(i=0;i<l;i++){
            if(i%2==0){
                c[j] = a[i];
                j++;
            }else{
                 b[k] = a[i];
                k++;
            }
            
      
        }
         c[j]='\0';
         b[k]='\0';
        cout<<c<<" "<<b<<endl;
    
        n--;
    }
    return 0;
}

